code/modules/reqs/supplypacks.dm

```diff
@@ -798,12 +796,8 @@ ARMOR
 	name = "Jaeger baldur modules"
 	contains = list(
 		/obj/item/armor_module/attachable/better_shoulder_lamp,
-		/obj/item/armor_module/attachable/better_shoulder_lamp,
-		/obj/item/armor_module/attachable/better_shoulder_lamp,
-		/obj/item/armor_module/attachable/better_shoulder_lamp,
-		/obj/item/armor_module/attachable/better_shoulder_lamp,
 	)
-	cost = 7
+	cost = 12
```

Comment:
12 supply points for a single baldur lamp? Personally I think you're overestimating the value of the lamp. Being (Nearly) fireproof or acid proof is well worth 12 req points.

Reply:
If I have a Baldur module and a raillight on my gun (or a beltgun that's not my primary...), I can see the WHOLE screen, minus one single tile at each corner of the screen.

Might even be able to do that without a raillight, TBH. But free Baldur is functionally equivalent to NVG's for everyone.